int32 id
int8 status
